core/cmd/remote_client.go

```diff
@@ -463,6 +463,15 @@ func (cli *Client) ConfigDump(c *clipkg.Context) (err error) {
 	return nil
 }
 
+func (cli *Client) ConfigFileValidate(c *clipkg.Context) error {
+	err := cli.Config.Validate()
+	if err != nil {
+		return err
+	}
+	cli.Config.LogConfiguration(func(params ...any) { fmt.Println(params...) })
```

Comment:
Does this work?
```suggestion
	cli.Config.LogConfiguration(fmt.Println)
```

Reply:
It does not. I might be missing something specific to golang or maybe it's because of the return type? Curious to know...

`cannot use fmt.Println (value of type func(a ...any) (n int, err error)) as "github.com/smartcontractkit/chainlink/core/config".LogFn value in argument to cli.Config.LogConfigurationcompiler[IncompatibleAssign](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal?utm_source%3Dgopls#IncompatibleAssign)`

Reply:
Yeah the return type, nbd